app/actions/CustomActionBuilders.scala

```diff
@@ -60,6 +65,16 @@ class CustomActionBuilders(
 
   val AuthenticatedAction = PrivateAction andThen authenticated()
 
+  val NewSignInFlowAuthenticatedAction = (useNewSignIn: Option[Boolean]) =>
```

Comment:
maybe `SignInFlowAuthenticatedAction` since this isn't necessarily the new flow?

Reply:
👍